src/engraving/dom/measurebase.cpp

```diff
@@ -950,10 +950,8 @@ void MeasureBaseList::change(MeasureBase* ob, MeasureBase* nb)
 
 void MeasureBaseList::append(MeasureBase* m)
 {
-    MeasureBase* next = m->next();
-    MeasureBase* prev = m->prev();
-    assert(!next);
-    assert(!prev || prev == m_last);
+    assert(!m->next());
+    assert(!m->prev() || m->prev() == m_last);
```

Comment:
Shouldn't we better use
```cpp
    IF_ASSERT_FAILED(m->next()) [
        return;
    }
    IF_ASSERT_FAILED(m->prev() && m->prev() != m_last) {
        return;
    }
```

Reply:
I don't think that's really necessary; the likelihood that that happens is low enough.

Reply:
Yeah, this situation should never happen in practice. The asserts are just to catch developers!